test/dkms_per_module_config_exclusive_test-1.0/dkms.conf

```diff
@@ -0,0 +1,17 @@
+PACKAGE_NAME="dkms_per_module_config_exclusive_test"
+PACKAGE_VERSION="1.0"
+
+# Two modules with different config requirements
+BUILT_MODULE_NAME[0]="dkms_per_module_config_exclusive_test_mod1"
+BUILT_MODULE_NAME[1]="dkms_per_module_config_exclusive_test_mod2"
+
+DEST_MODULE_LOCATION[0]="/updates/dkms"
+DEST_MODULE_LOCATION[1]="/updates/dkms"
+
+# Per-module config restrictions
+# Module 0: requires CONFIG_MODULES to be enabled (should always match)
+BUILD_EXCLUSIVE_CONFIG[0]="CONFIG_MODULES"
+# Module 1: requires CONFIG_NONEXISTENT_CONFIG to be enabled (should never match)
+BUILD_EXCLUSIVE_CONFIG[1]="CONFIG_NONEXISTENT_CONFIG"
```

Comment:
You should also add cases for `!CONFIG_MODULES` and `!CONFIG_NONEXISTENT_CONFIG` to test the negative cases

Reply:
Good point, will add it.

Reply:
Added both conditions in the same `BUILD_EXCLUSIVE_CONFIG`, so now there are two conditions for each directive (one true and one false).